weather_sp/splitter_pipeline/pipeline.py

```diff
@@ -117,35 +117,31 @@ def run(argv: t.List[str], save_main_session: bool = True):
 
     pipeline_options = PipelineOptions(pipeline_args)
     pipeline_options.view_as(SetupOptions).save_main_session = save_main_session
-    input_pattern = known_args.input_pattern
-    input_base_dir = _get_base_input_directory(input_pattern)
-    output_template = known_args.output_template
-    output_dir = known_args.output_dir
-    formatting = known_args.formatting
+    input_base_dir = _get_base_input_directory(known_args.input_pattern)
```

Comment:
These changes seem somewhat unrelated to the memory leak...

Reply:
Yes, that is right. I'm happy to revert them / move them to a different CL.